src/utils/hash.ts

```diff
@@ -20,6 +20,10 @@ export async function hashValue(value: string, dataType: string, jwt?: string) {
     });
   }
 
+  if(dataType == "file_name") {
```

Comment:
we could put this right on the context class so that it only has to do the check there.    gotta save those microseconds.

Reply:
right, java has this logic in the file entity when building the context.

Reply:
<img width="723" alt="Screen Shot 2020-08-27 at 12 13 51 PM" src="https://user-images.githubusercontent.com/35306917/91485008-f5679e80-e85e-11ea-89b4-edabbe55fdcf.png">